### DISCUSSION TITLE
Schema for Spacecraft Bus

### DISCUSSION THREAD

**Discussion by dselva80** [eose-tools-org/eose-api#10] [General]
Alex to put here what the spacecraft design algorithm outputs.

**Comment by Snaggerdoodle** [comment]
Current output is just the spacecraft mass and estimated dimensions, plus the mass of each subsystem. Future plans involve including a list of components in the spacecraft.

**Comment by Snaggerdoodle** [comment]
Here is an example JSON file for the cost estimation
{
    "satelliteDryMass": 10.599907296597706,
    "structureMass": 2.5636656703406997,
    "propulsionMass": 0.4431382356939476,
    "ADCSMass": 0.34600000000000003,
    "avionicsMass": 0.42374639179185125,
    "thermalMass": 0.6430351495441342,
    "EPSMass": 3.8184486533311484,
    "satelliteBOLPower": 45.906784686648685,
    "satDataRatePerOrbit": 102400.0,
    "lifetime": 5,
    "numPlanes": 1,
    "numSats": 1,
    "instruments": [
        {
            "trl": 9,
            "mass": 1.75,
            "avgPower": 1.75,
            "dataRate": 102400.0
        }
    ],
    "launchVehicle": {
        "height": 5.06,
        "diameter": 3.85,
        "cost": 45.0
    }
}
